contrib/index.html

```diff
@@ -0,0 +1,199 @@
+          </div>
+          <div class="page-section__inner first__eta">
+            <code>@include span-columns(4);</code>
+          </div>
+          <div class="page-section__inner first__theta">
+            <code>@include span-columns(8);<br>
+            @include omega();</code>
+          </div>
+          <div class="page-section__inner first__iota">
+            <code>@include span-columns(5);</code>
+          </div>
+          <div class="page-section__inner first__kappa">
+            <code>@include span-columns(7);<br>
+            @include omega();</code>
+          </div>
+          <div class="page-section__inner first__lambda">
+            <code>@include span-columns(6);</code>
+          </div>
+          <div class="page-section__inner first__mu">
+            <code>@include span-columns(6);<br>
+            @include omega();</code>
+          </div>
+        </section>
+        <h3>Nesting columns</h3>
+        <section class="page-section second">
+          <code>@include outer-container;</code>
+          <div class="page-section__inner second__alpha">
+            <code>@include span-columns(4);</code>
+          </div>
+          <div class="page-section__inner second__beta">
+            <code class="block">@include span-columns(8);</code>
+            <aside class="second__beta__alpha">
+              <code>@include span-columns(4 of 8);</code>
+            </aside>
+            <article class="second__beta__beta">
+              <code>@include span-columns(4 of 8);</code>
+            </article>
+          </div>
+        </section>
+        <h3>Table grid</h3>
+        <section class="page-section third">
+          <code>@include outer-container;</code>
+          <div class="page-section__inner third__dummy">
+            <code>@include fill-parent();<br>
+              @include row(table);</code>
+          </div>
+          <div class="page-section__inner third__alpha">
+            <aside class="third__alpha__alpha">
+              <code>
+                @include span-columns(4);<br>
+                @include pad();
+              </code>
+            </aside>
+            <article class="third__alpha__beta">
+              <code>
+                @include span-columns(8);<br>
+                @include reset-display;<br>
+              </code>
+            </article>
+          </div>
+        </section>
+        <h3>Shifting columns</h3>
+        <section class="page-section fourth">
+          <code>@include outer-container;</code>
+          <div class="page-section__inner fourth__alpha">
+            <code>@include span-columns(6);<br>
+              @include shift(3);</code>
+          </div>
+        </section>
+        <h3>Automatic rows</h3>
+        <div class="code">
+          <code>@include outer-container;</code>
+        </div>
+        <section class="page-section fifth">
+          <div class="box">
+<pre>
+  .box {
+    @include span-columns(3);
+    @include omega(4n);
+  }
+</pre>
```

Comment:
This part looks really strange centered, but I'm not sure I have a good solution.